.github/workflows/sonar.yml

```diff
@@ -0,0 +1,46 @@
+name: SonarQube analysis
+
+on:
+  pull_request:
+    types: [opened, synchronize]
+  push:
+    branches:
+      - main
+      - master
+      - staging
+      - qa
+      - development
+
+env:
+  SONAR_PROJECT: myapp-delete
```

Comment:
There's a `-Dsonar.projectKey` argument for the `sonar-scanner` CLI being used at line `43`, so is this environment variable used here or can it be deleted otherwise?

Reply:
Oops! Yesss my fault, should be deleted 😬